polaris/hub/client.py

```diff
@@ -866,10 +867,13 @@ def evaluate_competition(
         Returns:
              A `BenchmarkResults` object.
         """
-        return self._base_request_to_hub(
+        response = self._base_request_to_hub(
             url=f"/v2/competition/evaluate",
             method="PUT",
             json={
                 "competition": competition.artifact_id,
                 "predictions": y_pred
             })
+        return BenchmarkResults(results=pd.read_json(response["scores"]),
```

Comment:
I realized the hub can just return the scores, so we don't need to faff with constructing a valid competition object there. This still returns a `BenchmarkResults`, but now the hub just does the evaluation and nothing more.